.github/actions/reusable-python-setup_hatch/action.yml

```diff
@@ -0,0 +1,34 @@
+name: Setup hatch env
+description: Setup hatch environment for Python
+inputs:
+  cache_number:
+    description: 'Cache number to reset cache if pyproject.toml has not changed'
+    required: true
+    type: number
+  runner_os:
+    description: 'Runner OS'
+    required: true
+    type: string
+
+runs:
+  using: "composite"
+  steps:
+    - name: Get full Python version
+      id: full-python-version
+      run: echo "version=$(python -c "import sys; print('-'.join(str(v) for v in sys.version_info))")" >> $GITHUB_OUTPUT
+      shell: bash
+    - name: Set up cache
+      uses: actions/cache@v4
+      id: cache
+      env:
+        CACHE_NUMBER: ${{ inputs.cache_number }}
```

Comment:
**issue:** not used, as command line is using inputs.CACHE_NUMBER

should it use `inputs.cache_number` ? `env.CACHE_NUMBER` ? Is env needed at all if input value can be used